src/modify_menu_panel.js

```diff
@@ -6,25 +6,40 @@ function ModifyMenuPanel(editor) {
 	this.menus = {};
 	this.menuColumn = null;
 	this.menuContainer = null;
-	
+
 }
 
 ModifyMenuPanel.prototype.initialize = function (parentContainer) {
 	this.menuColumn = $("<div/>").attr('class', menuColumnClass).appendTo(parentContainer);
-	
+
 	// Generate the document status panel, which shows a save/export button as well as if there are changes to the document
 	if (this.editor.options.enableDocumentStatusPanel) {
-		var documentStatusPanel = $("<div>");
+		var self = this;
+		var documentStatusPanel = $(self.editor.options.documentStatusPanelDomId);
 		$("<span/>").addClass(submissionStatusClass).html("Document is unchanged")
 			.appendTo(documentStatusPanel);
-		var submitButton = $("<input/>").attr({
-			'id' : submitButtonClass,
-			'type' : 'button',
-			'class' : 'send_xml',
-			'name' : 'submit',
-			'value' : 'Submit Changes'
-		}).appendTo(documentStatusPanel);
-		if (this.editor.options.ajaxOptions.xmlUploadPath == null) {
+
+		if (self.editor.options.ajaxOptions.xmlUploadConfig != null){
+			$.each(self.editor.options.ajaxOptions.xmlUploadConfig, function(index, el){
```

Comment:
Yes, that makes sense to me.  I can think of situations at my own institution where having multiple submission destinations would be useful.  Or even just having a "save" versus "save and close" option